src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -153,6 +157,52 @@ public AndroidDriver(Capabilities desiredCapabilities) {
     }
 
     /**
+     * returns the information type of the system state which is supported to read
```

Comment:
Hi @heeseon 
Everything is ok but... could you remove the code below and move java doc to the `HasSupportedPerformanceDataType`? It is not necessary to duplicate code of API implemented by default. I think there is no specific details of the AndroidDriver behaviour.